src/mpo/mpo.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+# Define the Pauli matrices
+sigma_x = np.array([[0, 1], [1, 0]])
+sigma_y = np.array([[0, -1j], [1j, 0]])
+sigma_z = np.array([[1, 0], [0, -1]])
```

Comment:
These don't get used anywhere --> DELETE!

Reply:
Oh wait. I think they get used on line 32? If so, let's either:
(1) Move them into a new file called `_utils.py` and rename to CamelCase like `SigmaX` or
(2) Move them to within the function below

Reasoning: 
If they get used elsewhere (as in you think they will definitely be used elsewhere) then they should be put into a separate file that can get imported elsewhere. IF this is the case, they should be renamed to CamelCase to indicate they are constant objects (not to be modified). 
If they won't get used elsewhere, they should only live within the function that way they can't get imported (even accidentally) elsewhere.